### PR TITLE
Fix release.yml

### PR DESCRIPTION
**Pull request by pruivo**
I'm not sure if invoking "exit 1" would kill the release process or not.

### CODE REVIEW

**Comment by ryanemerson** [comment]
@pruivo Hmm not sure. I guess we could merge and then test it by setting `infinispanVersion` to include SNAPSHOT.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -57,11 +57,12 @@ jobs:
 
       - name: Tag release
         run: |
-          sed -i 's/<infinispan.version>.*<\/infinispan.version>/<infinispan.version>${{ github.event.inputs.infinispanVersion }}<\/infinispan.version>/g' pom.xml
+          sed -i '9s/<version>.*<\/version>/<version>${{ github.event.inputs.infinispanVersion }}<\/version>/' pom.xml
           find . -name "pom.xml" | xargs sed -i 's/<version>${{ github.event.inputs.currentVersion }}<\/version>/<version>${{ github.event.inputs.releaseVersion }}<\/version>/g'
+          grep SNAPSHOT --include pom.xml --recursive . || exit 1
```
Comment (ryanemerson) [inline .github/workflows/release.yml:62]: Don't we want this to fail only if grep matches? So wouldn't it be `&&`?

Comment (pruivo) [reply .github/workflows/release.yml]: nice catch! fixed!

**Comment by ryanemerson** [comment]
Thanks @pruivo

**Pull request merged**